server/routes/index.coffee

```diff
@@ -144,6 +144,7 @@ module.exports.setup = (app) ->
   app.get('/db/user', mw.users.fetchByGPlusID, mw.users.fetchByFacebookID)
   app.put('/db/user/-/become-student', mw.users.becomeStudent)
   app.put('/db/user/-/remain-teacher', mw.users.remainTeacher)
+  app.get('/db/user/-/lead-priority', mw.auth.checkLoggedIn(), mw.users.getLeadPriority)
```

Comment:
I think we should be moving away from 'hyphen' endpoints. Ideally, this endpoint takes a user handle and returns the priority for that user, and the user can get the priority for just themselves if they're not an admin. But, if there's not time to do this... ah well!